packages/translations/locales/en/dashboard.json

```
@@ -1338,7 +1435,17 @@
     "productMarketplace": "Product Marketplace",
     "productSuppliers": "Product Suppliers",
     "products": "Products",
+    "findCandidates": "Find candidates",
+    "findReplacements": "Find replacement staff",
+    "internPool": "Intern Pool",
```

Biome (2.4.15)

[error] The key internPool was already declared.

(lint/suspicious/noDuplicateObjectKeys)